shared/js/ui/views/grade-scorecard.es6.js

```diff
@@ -0,0 +1,18 @@
+const Parent = require('./sliding-subview.es6.js')
+
+function GradeScorecard (ops) {
+  this.model = ops.model
+  this.template = ops.template
+
+  Parent.call(this, ops)
+
+  this.setupClose()
+}
```

Comment:
I think you'll want to make sure you subscribe to site model changes and re-render this view/template dynamically as things like tracker numbers change/trickle in.